compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/builders/ExpressionHelpers.kt

```diff
@@ -211,11 +211,11 @@ fun IrBuilderWithScope.irEquals(arg1: IrExpression, arg2: IrExpression, origin:
         arg1, arg2
     )
 
+fun IrBuilderWithScope.irNot(arg: IrExpression, origin: IrStatementOrigin? = null) =
```

Comment:
There's already an irNot in LowerUtils.kt.

Reply:
Thanks, I've changed the code to use that one instead.